src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
It is good practice to await the termination of the ExecutorService after calling shutdown() to ensure that all tasks have completed execution before the program exits. Consider using awaitTermination() with an appropriate timeout.

Reply:
And again, this is the task requirement too.